diff -ur apache-couchdb-0.10.0/configure.ac mod/configure.ac
--- apache-couchdb-0.10.0/configure.ac	2009-09-30 05:04:15.000000000 +0200
+++ mod/configure.ac	2009-11-02 23:10:04.811654170 +0100
@@ -11,6 +11,7 @@
 dnl under the License.
 
 m4_include([m4/ac_check_icu.m4])
+m4_include([m4/ac_check_curl.m4])
 
 AC_INIT([LOCAL_PACKAGE_NAME], [LOCAL_VERSION], [], [LOCAL_PACKAGE_TARNAME])
 
@@ -18,6 +19,7 @@
 
 AC_CONFIG_SRCDIR([CHANGES])
 AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_MACRO_DIR([m4])
 
 AM_CONFIG_HEADER([config.h])
 
diff -ur apache-couchdb-0.10.0/Makefile.am mod/Makefile.am
--- apache-couchdb-0.10.0/Makefile.am	2009-10-08 06:17:52.000000000 +0200
+++ mod/Makefile.am	2009-11-02 23:10:29.278320255 +0100
@@ -12,6 +12,8 @@
 
 SUBDIRS = bin etc src share test var utils
 
+ACLOCAL_AMFLAGS = -I m4
+
 localdoc_DATA = AUTHORS.gz BUGS.gz CHANGES.gz NEWS.gz README.gz THANKS.gz
 
 DISTCLEANFILES = $(localdoc_DATA)
